config_repo/config.sh.repo

```diff
@@ -1,139 +1,187 @@
+# Adjust IMG_WIDTH and IMG_HEIGHT according to your camera's sensor ratio.
+IMG_RESIZE="false"
+IMG_WIDTH=2028
+IMG_HEIGHT=1520
 
-# Set to true to upload current image to your website
-UPLOAD_IMG=false
+# Crop images, used to improve them when using a fisheye lens.
+CROP_IMAGE="false"
+CROP_WIDTH=640
+CROP_HEIGHT=480
+CROP_OFFSET_X=0
+CROP_OFFSET_Y=0
+
+# Auto stretch images saved at night.
+AUTO_STRETCH="false"
+AUTO_STRETCH_AMOUNT=10
+AUTO_STRETCH_MID_POINT="10%"
 
-# Set to true to upload timelapse video to your website at the end of each night
-UPLOAD_VIDEO=false
+# Resize uploaded images.  Change the size to fit your sensor.
+RESIZE_UPLOADS="false"
+RESIZE_UPLOADS_SIZE="962x720"
+
+# Remove corrupt images before generating keograms and startrails.
+# This can take tens of minutes to run and isn't necessary unless your system produces
+# corrupt images which then generate funny colors in the summary images...
+REMOVE_BAD_IMAGES="false"
 
-# Set to true to send data to your server at the end of each night
-POST_END_OF_NIGHT_DATA=false
+# If REMOVE_BAD_IMAGES is "true", images whose mean brightness is
+# less than THRESHOLD_LOW or greater than THRESHOLD_HIGH percent (max: 100) will be removed.
+# Set to 0 to disable the brightness checks.
+REMOVE_BAD_IMAGES_THRESHOLD_LOW=1
+REMOVE_BAD_IMAGES_THRESHOLD_HIGH=90
 
-# Set to true to generate a timelapse at the end of the night
-TIMELAPSE=true
 
-# Set the width and height parameter for timelapse generation (sizes must be divisible by 2)
+########## Timelapse
+# Set to "true" to generate a timelapse video at the end of each night.
+TIMELAPSE="true"
+
+# Set the image width and height for timelapse video (sizes must be divisible by 2).
 TIMELAPSEWIDTH=0
 TIMELAPSEHEIGHT=0
 
-# Set the bitrate of the timelapse video.  Higher numbers will produce higher quality but larger files.
+# Bitrate of the timelapse video.  Higher numbers will produce higher quality but larger files.
 TIMELAPSE_BITRATE="2000k"
 
-# Timelapse frame rate (number of frames per second)
+# Timelapse video Frames Per Second.
 FPS=25
 
-# Encoder for timelapse. May be changed to use a hardware encoder or different codec
-VCODEC=libx264
-
-# Set to true to generate a keogram at the end of the night (image summary of the night)
-KEOGRAM=true
+# Encoder for timelapse video. May be changed to use a hardware encoder or different codec.
+VCODEC="libx264"
 
-# Set to true to upload the keogram after it has been generated
-UPLOAD_KEOGRAM=false
+# FFLOG determines the amount of log information displayed while creating a timelapse video.
+# Set to "info" to see additional information if you are tuning the algorithm.
+FFLOG="warning"
 
-# Set to true to generate a startrails image of the night. Will skip brighter images to avoid over-exposure
-STARTRAILS=true
+# While creating a timelapse video, a sequence of links to the images is created.
+# Set to "1" to keep that sequence; set to null ("") to have it deleted when done.
+KEEP_SEQUENCE=""
```

Comment:
Could you update the `KEEP_SEQUENCE` test in `timelapse.sh` to check for `true`, to be consistent with other settings?